src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
```

Comment:
The class name 'Drawable' is a verb, which is not recommended for class/interface names. Consider renaming it to something like 'DrawableFigure' to reflect that it's an entity capable of being drawn.